datadog_lambda/tracing.py

```diff
@@ -175,8 +174,9 @@ def extract_context_from_lambda_context(lambda_context):
             trace_id = client_context.custom.get(TraceHeader.TRACE_ID)
             parent_id = client_context.custom.get(TraceHeader.PARENT_ID)
             sampling_priority = client_context.custom.get(TraceHeader.SAMPLING_PRIORITY)
```

Comment:
Consider what if the `client_context.custom` has w3c headers, not just dd headers.